### ISSUE TITLE
Resnet10

### ISSUE THREAD

**Issue by cunjunyu** [facebookresearch/classifier-balancing#6]
Thanks for the excellent work!

I'm currently trying to reproduce the result of ResNet10. However, I find it hard to reproduce the same accuracy in the paper under the cRT setting using a similar config file of ResNext50.

Would you mind sharing the pre-trained model or config file? thx.

**Comment by bingykang** [comment]
What batch size are you using? If you are using a different batch size, please adjust the learning rate accordingly.

**Comment by cunjunyu** [comment]
Thanks for your prompt reply. After tuning some hyper-parameters, the result reaches the number reported in the paper. Thx

**Comment by dvirsamuel** [comment]
Hi @Majiker , can you please share the config file you used? Thanks 😄
